### DISCUSSION TITLE
Definicion del Proyecto Imposible

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Electronica-Microcontrolada/Grupo8#90] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by JorEl057** [comment]
Chicos, que tal como estan :

Manejo de Control de versiones: Ok
Manejo de Estructura de proyecto: Ok
Presentación Final: Falta presentar video con la demostración de lo hecho, la presentación grupal, subir código, por favor completar el proyecto antes del 31/10/22.

**Comment by JorEl057** [comment]
Devolución Grupal: @evera8  @MaviSchaf  @tecnosisnet  @Cabro645  @ProgramadorPegasus 
Nota Final Grupal: 6 (Seis)

Ideación del Proyecto: 7 (Siete)
Presentación: 6 (Seis)
Uso de recursos: 10 (Diez)
Desarrollo: 4 (Cuatro)
Implementación: 4 (Cuatro)

Nota Final Grupal: 6 (Seis)

Estimados tomaron la idea de Tratamiento de Efluentes Líquidos como proyecto y la ampliaron como un proyecto general de uso para reducción de consumo energético, aplicable a cualquier tipo de dispositivo de estas características. Esto demuestra su compromiso y RSU. En su presentación estuvieron un poco corto con el tiempo estipulado (Gantt), hablan sobre la necesidad en forma general y la implementación en forma particular. Trabajaron demostrando una organización y de recursos agiles muy bien logrados. Finalmente el desarrollo y la implementación del proyecto hubiese sido un poco más rico si se llegaba a plasmar en una pequeña maqueta.  Felicitaciones y continúen trabajando así!!!
